src/wp-includes/rest-api/endpoints/class-wp-rest-posts-controller.php

```diff
@@ -2172,6 +2176,14 @@ public function get_item_schema() {
 						'sanitize_callback' => array( $this, 'sanitize_slug' ),
 					),
 				),
+				'old_slug'         => array(
+					'description' => __( 'An alphanumeric identifier for the object unique to its type.' ),
+					'type'        => 'string',
```

Comment:
```suggestion
				'old_slug'         => array(
					'description' => __( 'The old alphanumeric identifier for the object.' ),
					'type'        => array( 'string', 'null' ),
```

Reply:
It might be better to allow `null` as a value for the old slug (when it is not set) and change the sanitiser to something like this:

```
/**
* Checks whether the old slug is valid.
*
* @param string $old_slug The old slug.
* @return string|null The sanitized old slug.
*/
public function sanitize_old_slug( $old_slug ) {
if ( is_null( $old_slug ) ) {
	return null;
} else {
	return $this->sanitize_slug( $old_slug );
}
}
```